### PR TITLE
pigz easyconfigs for goolf-1.4.10 and ictce-6.2.5

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

**Comment by fgeorgatos** [comment]
Test report by @fgeorgatos
**FAILED**
Build succeeded for 1 out of 10
Linux debian 6.0.10, Intel(R) Xeon(R) CPU           L5640  @ 2.27GHz, Python 2.6.6
See https://gist.github.com/a74d3b2d1b3d0a444f99 for a full test report.

File: easybuild/easyconfigs/p/pigz/pigz-2.3.1-goolf-1.4.10.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+easyblock = 'MakeCp'
+
+name = 'pigz'
+version = '2.3.1' 
+
+homepage = 'http://zlib.net/pigz/'
+description = """ pigz, which stands for parallel implementation of gzip, is a fully 
+ functional replacement for gzip that exploits multiple processors and multiple cores 
+ to the hilt when compressing data. pigz was written by Mark Adler, and uses the zlib 
+ and pthread libraries. """
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/p/pigz/pigz-2.3.1-goolf-1.4.10.eb]: yeap, you are right, that was me about the indentation. just keep doing what you do ;-)

**Comment by fgeorgatos** [comment]
Hi, please ignore my test of FAILED status of Feb15, it was just caused by a missing icc source (it won't come automatically unless someone feeds it by hand).

It would be nice to see this PR merged in, otherwise duplication of work is unavoidable. Point in case:

https://github.com/emi80/easyconfigs/blob/master/p/pigz/pigz-2.3.1-goolf-1.4.10-no-OFED.eb
(!)

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/9e5bd045ca85f9e17c11 for a full test report.

**Comment by boegel** [comment]
ignore the failed test, screwup on my end, retrying

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/356a89d3ae20ebe674fb for a full test report.

**Comment by boegel** [comment]
(finally) going in, thanks @pescobar and @fgeorgatos for review/testing

**Pull request merged**